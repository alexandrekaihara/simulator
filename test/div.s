LOAD TEN
DIV MINUSTWO
ADD FIVE
JMPZ ISZERO
OUTPUT MINUSONE
STOP
ISZERO: LOAD TEN
DIV TWO
SUB FIVE
JMPZ ISZERO2
OUTPUT MINUSONE
STOP
ISZERO2: OUTPUT ONE
STOP
ONE: CONST 1
MINUSONE: CONST -1
TEN: CONST 10
TWO: CONST 2
MINUSTWO: CONST -2
FIVE: CONST 5