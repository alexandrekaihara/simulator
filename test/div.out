10 27 4 29 1 30 8 11 13 26 14 10 27 4 28 2 30 8 22 13 26 14 13 25 14 1 -1 10 2 -2 5